opencti-platform/opencti-front/lang/back/en.json

```diff
@@ -168,6 +168,7 @@
   "Default marking": "Default marking",
   "Default time field": "Default time field",
   "Definition": "Definition",
+  "Delete this support package": "Delete this support package",
```

Comment:
Why this translation is in `back` folder? Should be only on `front` folder I guess

Reply:
Same for other languages

Reply:
you're right it's a mistake